las/client.py

```diff
@@ -558,9 +572,12 @@ def create_document(
         :raises: :py:class:`~las.InvalidCredentialsException`, :py:class:`~las.TooManyRequestsException`,\
  :py:class:`~las.LimitExceededException`, :py:class:`requests.exception.RequestException`
         """
+        find_content_type = not content_type
+        content_bytes, guessed_content_type = parse_content(content, find_content_type)
```

Comment:
Is there a point in not just always guess content type and rather determine how to use it based on 'content_type' arg?
For instance later 'contentType': content_type or guessed_content_type and maybe even print a warning if content_type != guessed_content_type?

```suggestion
        content_bytes, guessed_content_type = parse_content(content)
```

Reply:
It would make the function less efficient if we always guess content type, but it might not be as significant?

Reply:
I think the library looks at the first 10 bytes of the bytes provided. I can't imagine it being any significant?